source/PlayerInfo.h

```diff
@@ -181,6 +181,17 @@ class PlayerInfo {
 	// Get mission information.
 	const std::list<Mission> &Missions() const;
 	const std::list<Mission> &AvailableJobs() const;
+
+	enum SortType {ABC, PAY, SPEED, CONVENIENT};
```

Comment:
if you look at other enums in the code they are not declared alongside the methods; but in a separate section

Reply:
I had looked at enums in Mission.h `enum Location` and `enum Trigger`

Reply:
You're right it, it's not always at the front.